assets/backend/store/customers/customers.ts

```diff
@@ -1,40 +1,103 @@
+import { addQueryArgs } from '@wordpress/url';
 import { produce } from 'immer';
 import apiFetch, { APIResponse } from '~/backend/utils/fetch';
+import { Customer } from '~/backend/types';
 import { FetchFromApiActionReturn, baseActions } from '../actions';
 import { type State } from '../store';
 import { type CustomersState } from './customers.types';
 
 type Action = ReturnType<(typeof actions)[keyof typeof actions]>;
+type Query = Record<string, any>;
+type Response = APIResponse<{
+	customers: Customer[];
+	totalItems: number;
+	totalPages: number;
+	postsPerPage: number;
+	currentPage: number;
+}>;
 
 export const DEFAULT_CUSTOMERS_STATE: CustomersState = {
-	allCustomers: [],
+	customers: [],
+	totalItems: 0,
+	totalPages: 1,
+	postsPerPage: 10,
+	currentPage: 1,
 };
 
 export const actions = {
-	setAllCustomers(customers: State['customers']['allCustomers']) {
+	setCustomers(
+		customers: Customer[],
+		totalItems: number,
+		totalPages: number,
+		postsPerPage: number,
+		currentPage: number
+	) {
 		return {
 			type: 'SET_CUSTOMERS',
 			customers,
+			totalItems,
+			totalPages,
+			postsPerPage,
+			currentPage,
 		} as const;
 	},
```

Comment:
Same comment as with appointments. Can we use object instead of separate arguments?